### PR TITLE
[Doc Improvement][App notification for your app][3698849]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/709d7cedc446b84c0ca6c6ae6d3ab61ef8d34613/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e2b4e818-8562-4084-850b-4cde45afbe55) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/709d7cedc446b84c0ca6c6ae6d3ab61ef8d34613/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/080a2253-3008-44f1-9470-a7265cecb129) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/64d53154d7614de0dcde90673a23e6b596c772c5/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/53c1cad2-0c19-44db-9a09-ca484a70ecd9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64d53154d7614de0dcde90673a23e6b596c772c5/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/651ab37c-cd50-4ad2-8ef8-1f76aa02f3c1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b4602faa70dbe27397cfddaac1ebc1ef7a085841/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8f8606b0-ecfb-43e0-88f7-84f64805cc64) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b4602faa70dbe27397cfddaac1ebc1ef7a085841/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/97dc3b6f-5b8d-4e71-a960-8ece2d5bef6c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/74adb305d8d5724a815280dfa65fc15ebc512709/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa7a8c10-ae75-4d48-81d2-167888f75dbb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/74adb305d8d5724a815280dfa65fc15ebc512709/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/768b4dd4-4675-4cdb-952b-42a8bc133823) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f12b92f4015f79dfecb11f8c6d1b0bca6fdbf2fe/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3431b028-a7e6-4bc8-b3e6-502f3b1888b8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f12b92f4015f79dfecb11f8c6d1b0bca6fdbf2fe/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c0c85e74-f5e7-431f-8a0c-32c9bcb46834) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b49bf67a0c556a1c81c6bc16befce4e562b65ed0/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/15c20ac6-b102-4228-8e4e-b71f2d1fc679) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b49bf67a0c556a1c81c6bc16befce4e562b65ed0/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a417cee3-8295-4cad-b822-59dc962a7786) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/de08baed84695f9a0c5cf260f06230b83534108b/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0732db08-6714-479f-92f9-daffda7461b3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/de08baed84695f9a0c5cf260f06230b83534108b/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/882647d3-7b9b-444a-b8f8-e702d7d5ef86) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/5c02d10b1c0b05ec945ad39f127269cd113f23c4/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/159faf88-9756-4109-94ec-a8b07fb4892e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c02d10b1c0b05ec945ad39f127269cd113f23c4/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8dfaaae3-bbc0-4c3b-b52e-11847ff14b2b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c02d10b1c0b05ec945ad39f127269cd113f23c4/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/590e5b44-7163-4ac1-86a0-6e76a6d247dc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e645fc92ddb9c89a8c5db753145e93ab97677e9f/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c2913f85-c4fe-45a2-b281-5de4092b02ab) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e645fc92ddb9c89a8c5db753145e93ab97677e9f/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4f265181-944f-4de2-9310-2d073417b3d6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e645fc92ddb9c89a8c5db753145e93ab97677e9f/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/98cb1e18-03e4-40b3-8062-ca3bda178ef7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b98fddf3f2e375e195b770716f91d6ba756dd1b0/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/66bdad03-b583-4463-939b-13091245b326) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b98fddf3f2e375e195b770716f91d6ba756dd1b0/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/509c0235-560f-4780-b66a-5a04b6bb0f69) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b98fddf3f2e375e195b770716f91d6ba756dd1b0/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/780adf00-365c-4fff-a6b3-abb712514369) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/49170cb0f1626a6da4e621cf9201a8781af108b4/msteams-platform%2FTOC.yml) | 97 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a86a5b7-a190-4863-a852-531067a80e48) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/49170cb0f1626a6da4e621cf9201a8781af108b4/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22645bcd-d73e-4ee0-bd0e-fc1aeaec594f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/49170cb0f1626a6da4e621cf9201a8781af108b4/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/219c90b8-3d94-465a-baa7-eaba5c43c96f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/90c0cd023a65f211760dc49f10e59940d7402590/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4bcf24c8-a891-4033-a6e3-724bb3cc8b46) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/90c0cd023a65f211760dc49f10e59940d7402590/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a88cc9bf-4ba5-4403-9715-f0258f1d681a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/90c0cd023a65f211760dc49f10e59940d7402590/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8d09d82e-d40c-4455-8b73-aa8135148917) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/56299934cc6a07db2bc65c74d3add3a9a6a3ed25/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a018736d-3d82-41c8-8c32-3678699b8508) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/56299934cc6a07db2bc65c74d3add3a9a6a3ed25/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/658e9cba-3125-4bf5-ac5b-9ace9442358d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/56299934cc6a07db2bc65c74d3add3a9a6a3ed25/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/91ca7112-fe7f-4e77-9f89-8cbdeae161d9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/design/design-app-notification.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+* Use interactive notification bot when you want users to interact with Adaptive Card.
+
+How do you expect users to interact with the notification?
+
+* Use activity feed notifications, when there's no further conversation is expected between the user and bot.
+* Use interactive notification bot to encourage conversations with the bot.
+
+:::image type="content" source="../../assets/images/app-notifications.png" alt-text="Screenshot shows you to choose your right notification framework for your app." lightbox="../../assets/images/app-notifications.png":::
+
+Here's some example scenario's and recommended notification types:
```
Comment (aosolis) [inline msteams-platform/concepts/design/design-app-notification.md:81]: "scenarios" not "scenario's"

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/d0b132867ae81dea217ecd92c60d0a52378f8da3/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a687f5bb-6ecd-4672-86ab-dbdf14ed4cbe) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d0b132867ae81dea217ecd92c60d0a52378f8da3/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a1b77cd7-d0f5-49c8-b584-b083b0ac232d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d0b132867ae81dea217ecd92c60d0a52378f8da3/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a950052-99e7-4de9-aa98-9b46aa92b085) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/19f538fdeefc1c3a8dde018da95653cd0b90a0e9/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3b89dfca-eca3-453a-80b4-70d27ca39eee) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/19f538fdeefc1c3a8dde018da95653cd0b90a0e9/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8318aa73-6abb-41a0-860f-3bbcca7ca92e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/19f538fdeefc1c3a8dde018da95653cd0b90a0e9/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1fb3d136-187a-4d8d-8446-2149ed936e69) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2d28d8e8952ecc9d792d7f05d6357c8d89440489/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d2830ea9-5eb5-4500-929f-c9f7142fe231) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2d28d8e8952ecc9d792d7f05d6357c8d89440489/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f318b0f3-473e-4fad-9e4c-34d33b2e840d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2d28d8e8952ecc9d792d7f05d6357c8d89440489/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7dd2f406-bee0-4f88-a840-7488ec5b1efc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/design/design-app-notification.md (hunk 1 above)
Comment (v-sdhakshina) [reply msteams-platform/concepts/design/design-app-notification.md]: Updated as suggested.

**Review by luywang — APPROVED** [review]
Looks good. Approved.

**Comment by Carter425** [comment]
This all looks good to me as well

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/c81b5169b48cbbb61d514fa6360e18765b11d1ea/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/079d4246-bfb8-4625-881c-61263fbe7947) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/interactive-notification-bot-in-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c81b5169b48cbbb61d514fa6360e18765b11d1ea/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Finteractive-notification-bot-in-teams.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c7a8e2a4-57e8-4b96-b492-7c4615b9a699) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-app-notification.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c81b5169b48cbbb61d514fa6360e18765b11d1ea/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-app-notification.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/69db51de-9019-4c0b-8fa5-a19d9d53f8f3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**